dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -130,15 +130,21 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
     bool PushNotificationManager::IsSupported()
     {
-        // Elevated processes are not supported by PushNotifications. UnpackagedAppScenario is not supported when it is self contained
-        // because the PushNotificationsLongRunningProcess is unavailable due to missing the Singleton package.
-        static bool isSupported{ !Security::IntegrityLevel::IsElevated() && (PushNotificationHelpers::IsPackagedAppScenario() || !WindowsAppRuntime::SelfContained::IsSelfContained()) };
+        // UnpackagedAppScenario is not supported when it is self contained due to missing the Singleton package.
+        static bool isSupported{ PushNotificationHelpers::IsPackagedAppScenario() || !WindowsAppRuntime::SelfContained::IsSelfContained() };
         return isSupported;
 
     }
 
     winrt::Microsoft::Windows::PushNotifications::PushNotificationManager PushNotificationManager::Default()
     {
+        static std::once_flag flag;
+        std::call_once(flag, []()
+        {
+            // Need to clear the RoActivateInstance caching for LRP proxyStub to be found.
+            PushNotificationHelpers::ClearRoActivateInstanceCache();
```

Comment:
SAme comment here. No need of the call_once lambda. Just move this to the else clause.